### PR TITLE
Embed Disqus

### LINKED ISSUE #371: Integrate Disqus
Each section should has a conversation in the footer, so we can embed Disqus in the current layout.

### PR DESCRIPTION
**Pull request by rafaparadela**
This PR embeds conversation of Disqus in each section footer.

@dialelo @raulraja Could you review please? 

Closes #371

### CODE REVIEW

**Comment by rafaparadela** [comment]
By the way, it seems like `preview sites` can't run properly:

```
2016-06-07T18:17:16.095251+00:00 heroku[web.1]: Error R14 (Memory quota exceeded)
2016-06-07T18:17:16.095251+00:00 heroku[web.1]: Process running mem=961M(187.8%)
```
I have set manually `JAVA_TOOL_OPTIONS` env var, but it still fails.

https://scala-exercises-pr-372.herokuapp.com/

File: server/app/views/templates/library/disqus.scala.html [hunk 1]
```diff
@@ -0,0 +1,21 @@
+@(path: String)
+<div class="comments">
+
+  <div id="disqus_thread"></div>
+  <script>
+       var disqus_config = function () {
+          this.page.url = 'https://scala-exercises.herokuapp.com@path';
```
Comment (raulraja) [inline server/app/views/templates/library/disqus.scala.html]: can't the host be inferred from the current browser location?

File: server/app/views/templates/library/disqus.scala.html [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+          this.page.identifier = '@path';
+       };
+
+      (function() { // DON'T EDIT BELOW THIS LINE
+        var d = document, s = d.createElement('script');
+        s.src = '//scala-exercises.disqus.com/embed.js';
```
Comment (raulraja) [inline server/app/views/templates/library/disqus.scala.html]: This should be an config setting

**Comment by rafaparadela** [comment]
Addressed comments, now data is being retrieved:

- domain and path from javascript `window.location`
- disqus domain from `application.conf`

**Comment by dialelo** [comment]
:+1:

**Comment by dialelo** [comment]
:shipit:

**Pull request merged**